src/config/constants/common.ts

```diff
@@ -84,6 +84,10 @@ export const TOKEN_BLACKLIST = [
   '0x48e345cb84895eab4db4c44ff9b619ca0be671d9', // ETH - WBTC. Euler. Disabled due to hack.
   '0xb95e6eee428902c234855990e18a632fa34407dc', // ETH - LUSD. Euler. Disabled due to hack.
   '0x7c6d161b367ec0605260628c37b8dd778446256b', // ETH - wstETH. Euler. Disabled due to hack.
+
+  '0xfa02ce0440dc377becc24d376750e5b1edcc8f42', // POLY - MATIcX Aave v3
+  '0xfa68fb4628dff1028cfec22b4162fccd0d45efb6', // POLY - MaticX
```

Comment:
No need to blacklist MaticX
```suggestion
  '0xfa02ce0440dc377becc24d376750e5b1edcc8f42', // POLY - MaticX Aave v3
```